site/src/routes/faq/rss.xml.js

```diff
@@ -0,0 +1,42 @@
+import send from '@polka/send';
```

Comment:
I'm guessing this file was inadvertently copied over from the blog and should be deleted.

Reply:
Whoops yeah :)